ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/GIT_Source/tasks/main.yml

```diff
@@ -12,6 +12,16 @@
   shell: git --version >/dev/null 2>&1
   ignore_errors: yes
   register: git_installed
+  when:
+    - ansible_distribution != "FreeBSD"
+  tags: git_source
+
+- name: Test if git is installed on path FreeBSD
+  shell: git --version >/dev/null
```

Comment:
@bblondin `git --version >/dev/null 2>&1` didn't work on FreeBSD so I had to create a separate check that runs `git --version >/dev/null`. I wasn't sure if the latter command would work on all systems in which case we could just update the initial check command

Reply:
This was for use of the conditions. Have you test all scenario
not install, install but wrong version, install and right version